blacklist-call/.env

```diff
@@ -0,0 +1,3 @@
+# The array of numbers you want to reject
+# List all blocked phone numbers in quotes and E.164 formatting, separated by a comma
+BLACKLIST=[ "+12125551234", "+17025556789" ]
```

Comment:
This should be a comma separated list instead. So:
`BLACKLIST="+12125551234,+17025556789"